cobra/__init__.py

```diff
@@ -194,9 +194,11 @@ def __init__(self, socket, sflags=0):
                 raise Exception('Missing "msgpack" python module ( http://visi.kenshoto.com/viki/Msgpack )')
 
             def msgpackloads(b):
+                logger.debug("<< %r  (%r)", b, loadargs)
                 return msgpack.loads(b, **loadargs)
 
             def msgpackdumps(b):
+                logger.debug(">> %r  (%r)", b, dumpargs)
```

Comment:
I'm going to be very confused if I every hit these messages. Can you give more context here? Even just the function name of where we are.

Reply:
the function name is part of the logging format.  ;)
i don't want to waste screen-real-estate printing it twice.

i can get rid of the messages altogether, but they're helpful for troubleshooting communications.